### PR TITLE
data:Pallavi-33

### PR DESCRIPTION
**Pull request by Pallavi-33**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5886"></a>

### CODE REVIEW

**Comment by Pallavi-33** [comment]
This file is merged into the original one ?

**Review by imbeshat — APPROVED** [review]

**Comment by imbeshat** [comment]
> This file is merged into the original one ?

Maintainers will merge it after approving.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**
